jira_agile_metrics/querymanager.py

```diff
@@ -8,6 +8,21 @@
 
 logger = logging.getLogger(__name__)
 
+def multi_getattr(obj, attr, **kw):
```

Comment:
This function allows us to lookup an object using notation as follows 

Given the input
```
issue: {
    team: {
        name: "Mobile"
    }
}
```

we can reference the team name using ...
```
multi_getattr(issue, "team.name")
```